src/sonic-config-engine/tests/sample_output/zebra_frr.conf

```diff
@@ -27,17 +29,4 @@ link-detect
 interface PortChannel04
 link-detect
 !
-!
-! Set ip source to loopback for bgp learned routes
```

Comment:
later you will add another pr to validate this, i assume.

Reply:
Do you mean test?

Reply:
yes